pkg/runner/mock_instance.go

```diff
@@ -0,0 +1,28 @@
+package runner
+
```

Comment:
What is the level of ``instance``

Reply:
I mean it's a function process or it's a pod

Reply:
It's a process